docs/config.md

```diff
@@ -28,4 +26,281 @@ order = 10
 name= "WorkflowPendingUserAuthnHandler"
 subscriptions =["PRE_AUTHENTICATION"]
 
```

Comment:
we need to ask the user to add the following config as well. this will enable workflow API in the console

```
[[api_resources]]
name = "workflow Approval API"
identifier = "/me/approval-tasks"
requiresAuthorization = true
type = "TENANT"
description = "API representation of the workflow Approval API"

[[api_resources.scopes]]
displayName = "View Workflow Approvals"
name = "internal_humantask_view"

```